### PR TITLE
Dark theme toggler

### PR DESCRIPTION
**Pull request by PuneetP16**
image

### CODE REVIEW

**Comment by sushmita12345** [comment]
Clean code and smooth UI, looks good to merge.

**Review by sushmita12345 — APPROVED** [review]

File: javascript/darkmodetoggler.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	console.log("Clicked");
+	if (darkMode !== "enabled") {
+		enableDarkMode();
+		e.currentTarget.innerHTML = '<i class="bx bxs-sun"></i>';
+	} else {
+		disableDarkMode();
+		e.currentTarget.innerHTML = '<i class="bx bxs-moon"></i>';
+	}
+};
+
```
Comment (tarunsankhla) [inline javascript/darkmodetoggler.js:37]: Add comments in js file

**Review by tarunsankhla — APPROVED** [review]
Over all looks good classnames and spacing

**Review by m26unkwn — APPROVED** [review]

**Comment by m26unkwn** [comment]
Things that I Think you have done well

1. Naming your css Classes
2. Transition animation on buttons 
3. Dark UI ❤

Code quality and structure improved.

File: javascript/darkmodetoggler.js (hunk 1 above)
Comment (PuneetP16) [reply javascript/darkmodetoggler.js:37]: Ok, I will look into that. Thanks for suggestion.

**Pull request merged**